libraries/ESP8266WiFi/examples/WiFiSmartConfig/WiFiSmartConfig.ino

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Smart config example sketch
+ *
+ */
```

Comment:
Please add:

- a statement that the sketch is in public domain or CC0 licensed
- instructions to use this example, like what kind of APP needs to be installed, what is the procedure to use this, and so on.